mapswipe_workers/tests/unittests/fixtures/build_area/tutorial.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
I'm  not sure if I would place the data for tutorlals here under `tests/unittests/fixtures` since it is also needed during the regular creation of tutorials. Any suggestions where it could fit better?